Makefile

```diff
@@ -21,9 +21,9 @@ Supported targets: 'develop', 'docs', 'sdist', 'clean', 'test', 'pypi', or 'pypi
 
 The 'develop' target creates an editable install (aka develop mode). Set the 'extras' variable to
 ensure that develop mode installs support for extras. Consult setup.py for a list of supported
-extras. For example, to install Toil in develop mode with Mesos, AWS and Azure support, run
+extras. For example, to install Toil in develop mode with Mesos, AWS, Azure and CWL support, run
 
-make develop extras=[mesos,aws,azure]
```

Comment:
The README now mentions the extras. I'm inclined to remove this example from the Makefile as it would be yet another place to modify if you add an extra to Toil.